Base/usr/share/man/man1/su.md

```diff
@@ -1,41 +1,42 @@
 ## Name
 
-su - run a command with substitute user
+su - switch to another user
 
 ## Synopsis
 
-```
-$ su [user]
+```**sh
```

Comment:
What's up with the \*\* here?

Reply:
Please familiarize yourself with the existing man pages, this is in literally all of them :^)

https://github.com/SerenityOS/serenity/blob/5d219ad4f7c342a9c23dfcc81e1d0be709fb3e46/Userland/Libraries/LibMarkdown/CodeBlock.cpp#L90-L100